### PR TITLE
20487D | Mod06 | LM

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Comments inline

### CODE REVIEW

File: Instructions/20487D_MOD06_LAB_MANUAL.md [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
 
-# Lab: Deploying an ASP.NET Core web service on Linux
+# Lab: Deploying an ASP.NET Core Web Service on Linux
 
 #### Scenario
 
-In this exercise you will deploy an ASP.NET Core web service on Linux
+In this exercise, you will deploy an ASP.NET Core web service on Linux
 
 #### Objectives
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_LAB_MANUAL.md]: Missing objectives

File: Instructions/20487D_MOD06_LAB_MANUAL.md [hunk 2]
```diff
@@ -1,54 +1,53 @@
...
 #### Task 1: Use Nginx in a container to reverse-proxy the ASP.NET web service
 
-1. Open **BlueYonder.Flights.Service** project in **VSCode**.
-2. Add New **nginx** folder .
-3. Add new **nginx.conf** file in the **nginx** folder.
-4. Paste the following code:
+1. Open the **BlueYonder.Flights.Service** project in Visual Studio Code.
+2. Add a new **nginx** folder.
+3. In the **nginx** folder, add a new **nginx.conf** file.
+4. In the new file, paste the following code:
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_LAB_MANUAL.md]: Please verify my edit here.

File: Instructions/20487D_MOD06_LAB_MANUAL.md [hunk 3]
```diff
@@ -73,11 +72,11 @@ After completing this lab, you will be able to:
    }
    ```
 
-   check **proxy_pass** this is the path of the revers proxy
-   and the **test** is what need to be added for using **proxy_pass**
+   check **proxy_pass** this is the path of the reverse proxy
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_LAB_MANUAL.md]: Should this and the next line be notes?

File: Instructions/20487D_MOD06_LAB_MANUAL.md [hunk 4]
```diff
@@ -120,66 +119,65 @@ After completing this lab, you will be able to:
...
+#### Task 1: Create a standard web app
+
+1. Open the Microsoft Azure Portal.
+2. Display all the app services.
+   - To select the app service template, in the **App Services** blade, click **Add**.
+   - In the **Web** blade, click **Web App**. An overview of the template will be shown. Click **Create**.
+3. Create the new **Web App** by entering details in the following fields:
+   - In the **App Name** box, enter the following web app name: **blueyonder-flights**-*{YourInitials}*
+   - In the **Resource Group** box, select **Create new** and then type **Mod6Resource**.
+   - **Create new** **App Service plan/Location**, then open the **New App Service Plan** blade, and enter the following information:
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_LAB_MANUAL.md]: Instruction is not clear. Please rewrite.

File: Instructions/20487D_MOD06_LAB_MANUAL.md [hunk 5]
```diff
@@ -247,185 +245,193 @@ After completing this lab, you will be able to:
...
-1. Click on **Destinations** in **BlueYonder** API.
-2. Click on **Test** tab in **Destinations** operation blade.
-3. Click on **Send** to test the **GET Destinations** API.
-4. Verified that the response is 200 with destinations list.
-5. Click on **Destinations By ID** in **BlueYonder** API.
-6. Click on **Test** tab in **Destinations By ID** operation blade.
-7. Type **1** under **VALUE** in the **id** parameter in the **Template parameters** section.  
-8. Click on **Send** to test the **GET Destinations by ID** API.
-9. Verified that the response is 200 with **Seattle** destination.
+1. To add new products, in **API MANAGEMENT**, on the top menu bar, click **Add**, and then enter:
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_LAB_MANUAL.md]: Please verify my edits.

File: Instructions/20487D_MOD06_LAB_MANUAL.md (hunk 3 above)
Comment (slavBarkal) [reply Instructions/20487D_MOD06_LAB_MANUAL.md]: yes

File: Instructions/20487D_MOD06_LAB_MANUAL.md (hunk 1 above)
Comment (AviAvni) [reply Instructions/20487D_MOD06_LAB_MANUAL.md]: I'v added the objectives

File: Instructions/20487D_MOD06_LAB_MANUAL.md (hunk 2 above)
Comment (AviAvni) [reply Instructions/20487D_MOD06_LAB_MANUAL.md]: ok

File: Instructions/20487D_MOD06_LAB_MANUAL.md (hunk 1 above)
Comment (barucht) [reply Instructions/20487D_MOD06_LAB_MANUAL.md]: @VinayApposite - see the objectives below, starting with "Deploy an ASP. Core web..." ending with "test the API with cahce rules"

**Pull request merged**